CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 ##### Bug Fixes
 
-* None.  
+* Fixed crash with very long devices list/slow computers. [Martin Fiebig](https://github.com/mfiebig) [#17](https://github.com/CocoaPods/fourflusher/issues/17)
```

Comment:
nit contributor should be in separate line and also add two empty spaces after `.` for formatting.

Reply:
Sorry about that. Should be ok now? @dnkoutso